rxgroups/src/main/java/com/airbnb/rxgroups/ProcessorUtils.java

```diff
@@ -0,0 +1,59 @@
+package com.airbnb.rxgroups;
```

Comment:
good point. Instead of a new project though, can we use `compileOnly` for those dependencies? https://blog.gradle.org/introducing-compile-only-dependencies